concord-server/server/src/main/java/com/rabidgremlin/concord/resources/PhrasesResource.java

```diff
@@ -254,4 +254,17 @@ public Response purgeVotesForPhrase(@ApiParam(hidden = true) @Auth Caller caller
     return Response.ok().build();
   }
 
+  @DELETE
+  @Path("/{phraseId}/delete")
+  @Timed
+  public Response deleteVote(@ApiParam(hidden = true) @Auth Caller caller, @PathParam("phraseId") String phraseId)
+  {
+    String userId = caller.getToken();
+    log.info("{} deleting user - {}'s vote for phrase {}", caller, userId, phraseId);
+
+    votesDao.deleteVote(phraseId, userId);
+
+    return Response.ok().build();
```

Comment:
Better if we make it return HTTP 204, because we are returning empty response here.  You can find how to do that from one of the POST methods